test/lib/compile/openapi.test.js

```diff
@@ -424,4 +424,47 @@ service CatalogService {
         expect(openAPI.externalDocs.url).toBe('https://help.sap.com/docs/product/123.html');
   }
   );
+
+  test('OpenAPI annotations: @OpenAPI.Extensions annotation is added to the openapi document', () => {
+    const csn = cds.compile.to.csn(`
+      namespace sap.OpenAPI.test;
+      @OpenAPI.Extensions: {
+        ![compliance-level]: 'sap:base:v1',
+        ![x-sap-ext-overview]: {
```

Comment:
shouldn't we add an `expect-toBe` check for `x-sap-ext-overview` nested value also?